messages/src/message.rs

```diff
@@ -108,6 +108,24 @@ pub enum HostRequest {
     /// Request that the modules be reset.
     Reset,
 
+    /// Request that the ResetL line be asserted.
+    AssertReset,
+
+    /// Request that the ResetL line be de-asserted.
+    DeassertReset,
+
+    /// Request that the LpMode line be asserted.
+    AssertLpMode,
+
+    /// Request that the LpMode line be de-asserted.
+    DeassertLpMode,
+
+    /// Request that power be enabled to a module should it be inserted.
+    EnablePower,
+
+    /// Request that power be disabled to a module should it be inserted.
+    DisablePower,
+
```

Comment:
I also can't comment on the relevant line, but I think we'd want to remove the `SetPowerMode` variant entirely from the protocol. The host-side and controller can expose that idea, and then control either the efuse, LPMode pin, or the correct bytes of the module memory map depending on which mode is requested and whether the module responds to LPMode.

I think it's enough to remove this variant; move the `PowerMode` enum to `controller/src/lib.rs`, and then maybe replace the body of the `Controller::set_power()` function with a `todo!()`. The implementation of that is going to change significantly to use these new messages, which seems better in a separate PR.

Reply:
I'm cool with removing `SetPowerMode`. It felt like a temporary, parallel path anyway.